### PR TITLE
Split User.elm to separate files 

### PR DESCRIPTION
**Pull request by HelenaEksler**
#102

### CODE REVIEW

File: src/elm/Pages/User/Decoder.elm [hunk 1]
```diff
@@ -0,0 +1,39 @@
+module Pages.User.Decoder where
+
+import Company exposing (..)
```
Comment (amitaibu) [inline src/elm/Pages/User/Decoder.elm]: order by alphabet

File: src/elm/Pages/User/Model.elm [hunk 2]
```diff
@@ -0,0 +1,55 @@
+module Pages.User.Model where
+
+import Company exposing (..)
+import Effects exposing (Effects)
+import Http exposing (Error)
+import String
+import Task
+
+import Debug
```
Comment (amitaibu) [inline src/elm/Pages/User/Model.elm]: remove

File: src/elm/Pages/User/Model.elm [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  }
+
+init : (Model, Effects Action)
+init =
+  ( initialModel
+  , Effects.none
+  )
+
+type Action
+  = GetDataFromServer
```
Comment (amitaibu) [inline src/elm/Pages/User/Model.elm]: order by alphabet

File: src/elm/Pages/User/Model.elm [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  | SetAccessToken AccessToken
+  | UpdateDataFromServer (Result Http.Error (Id, String, List Company.Model))
+  -- NoOp actions
```
Comment (amitaibu) [inline src/elm/Pages/User/Model.elm]: remove comment

File: src/elm/Pages/User/Update.elm [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+    SetAccessToken accessToken ->
+      ( {model | accessToken <- accessToken}
+      , Effects.none
+      )
+
+
+-- Determines if a call to the server should be done, based on having an access
+-- token present.
+isAccessTokenInStorage : Result err String -> Bool
```
Comment (amitaibu) [inline src/elm/Pages/User/Update.elm]: is this being used?

**Comment by amitaibu** [comment]
Looks good. I've added a few comments which are not related to the PR, that just popped up

File: src/elm/Pages/User/Update.elm (hunk 5 above)
Comment (HelenaEksler) [inline src/elm/Pages/User/Update.elm]: No, it's not. Remove?

Comment (amitaibu) [inline src/elm/Pages/User/Update.elm]: yes, please

Comment (amitaibu) [inline src/elm/Pages/User/Update.elm]: just a good time for some cleanup, is all :)

**Comment by HelenaEksler** [comment]
@amitaibu Done.

**Pull request merged**